library/std/unstable_arithmetic_internal.qs

```diff
@@ -323,12 +322,12 @@ namespace Microsoft.Quantum.Unstable.Arithmetic {
     /// Computes carries for the look-ahead adder
     internal operation ComputeCarries(ps : Qubit[], gs : Qubit[]) : Unit is Adj {
         let n = Length(gs);
-        Fact(Length(ps)+1 == n, "Register gs must be one qubit longer than register gs.");
+        Fact(Length(ps) + 1 == n, "Register gs must be one qubit longer than register gs.");
 
         let T = Floor(Lg(IntAsDouble(n)));
         use qs = Qubit[n - HammingWeightI(n) - T];
 
-        let registerPartition = MappedOverRange(t -> Floor(IntAsDouble(n) / IntAsDouble(2^t)) - 1, 1..T - 1);
+        let registerPartition = MappedOverRange(t -> Floor(IntAsDouble(n) / IntAsDouble(2 ^ t)) - 1, 1..T - 1);
```

Comment:
Hmmmm, I wonder if we should put spaces around exponentiation operators. That one does look a little weird.

Reply:
I asked other people what they prefer. I can special-case the logic for the carrot character to have no spaces, if we really want.

Reply:
Indentation logic will be addressed by a future PR.

Reply:
Let's keep this spacing for now. If people really complain about this, it is an easy one to update later.